packages/client/src/api/index.js

```diff
@@ -26,13 +26,14 @@ export const getSetList = async () => {
         const { data } = await response.json();
         let mastersList = ['uplist', 'slx', 'klr', 'plist', 'akr', 'fmb1', 'mb1', 'tpr', 'vma', 'me4', 'me3', 'me2', 'me1', 'ren', 'rin'];
         let falseSets = ['tscd', 'j21', 'h1r', 'tsb', '4bb', 'sum', 'fbb', 'jmp', 'j22', 'dbl', 'clb', 'cmr'];
-        let boosterSets = data.map(({ code, name, set_type, released_at }) => {
+        // eslint-disable-next-line
+        let boosterSets = data.map(({ code, name, set_type, released_at, icon_svg_uri }) => {
             if (set_type === 'core' || set_type === 'expansion' || set_type === 'draft_innovation') {
                 if (!falseSets.includes(code)) {
-                    return { code, name, set_type, release_date: released_at }
+                    return { code, name, set_type, release_date: released_at, icon_svg_uri }
                 }
             } else if (set_type === 'masters' && !mastersList.includes(code)) {
-                return { code, name, set_type, release_date: released_at }
+                return { code, name, set_type, release_date: released_at, icon_svg_uri }
```

Comment:
Just including the icon in the return for the set list screen. could maybe return whole obj?